dll/kernel32.cpp

```diff
@@ -613,25 +697,47 @@ namespace kernel32 {
 	}
 
 	unsigned int WIN_FUNC GetCurrentDirectoryA(unsigned int uSize, char *lpBuffer) {
-        DEBUG_LOG("GetCurrentDirectoryA\n");
+		DEBUG_LOG("GetCurrentDirectoryA\n");
 
-        std::filesystem::path cwd = std::filesystem::current_path();
+		std::filesystem::path cwd = std::filesystem::current_path();
 		std::string path = files::pathToWindows(cwd);
 
 		assert(path.size() < uSize);
 
-        strcpy(lpBuffer, path.c_str());
-        return path.size();
-    }
+		strcpy(lpBuffer, path.c_str());
+		return path.size();
+	}
 
 	void* WIN_FUNC GetModuleHandleA(const char* lpModuleName) {
-		DEBUG_LOG("GetModuleHandleA %s\n", lpModuleName);
+		DEBUG_LOG("GetModuleHandleA\n");
```

Comment:
the module name does seem useful to have?

Reply:
woops!